helm/values.yaml

```diff
@@ -0,0 +1,105 @@
+# Default values for astral.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/gr-oss-devops/astral
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+podLabels: {}
+
+podSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+
+securityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 3000
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: astral.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: astral-tls
+  #    hosts:
+  #      - astral.local
+
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+env:
+  - name: SECRET_KEY_BASE
+    value: secret
+  - name: RAILS_ENV
+    value: "production"
+  - name: RAILS_ROOT
+    value: "/app"
+  - name: BUNDLE_DEPLOYMENT
+    value: "1"
+  - name: BUNDLE_APP_CONFIG
+    value: "/app/.bundle"
+  - name: BUNDLE_WITHOUT
+    value: "test development"
+  - name: VAULT_ADDR
+    value: "XXX"
+  - name: VAULT_TOKEN
+    value: "XXX"
+  - name: JWT_SIGNING_KEY
+    value: "XXX"
+  - name: SHARED_VAULT_CERT_PATH
```

Comment:
I don't think we need the "SHARED" prefix on these two -- that section grouping is just a helper to prevent repeating in each environment type, but here there is only one environment class (production).